var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -295,8 +295,8 @@ def configure_args(self):
             config_args.extend(['--enable-mpi-cxx'])
 
         # Fabrics and schedulers
-        config_args.extend(self.with_or_without('fabrics'))
-        config_args.extend(self.with_or_without('schedulers'))
+        config_args += self.with_or_without('fabrics')
```

Comment:
What is the reason to prefer `+=` to `extend`? All changes in this file appear cosmetic - am I right about that?